src/python/gem5/resources/client_api/client_wrapper.py

```diff
@@ -289,15 +289,16 @@ def _check_resource_version_compatibility(
                 [resource], gem5_version=gem5_version
             )
         ):
-            warn(
-                f"Resource {resource['id']} with version "
-                f"{resource['resource_version']} is not known to be compatible"
-                f" with gem5 version {gem5_version}. "
-                "This may cause problems with your simulation. "
-                "This resource's compatibility "
-                "with different gem5 versions can be found here: "
-                "https://resources.gem5.org"
-                f"/resources/{resource['id']}/versions"
-            )
+            if not gem5_version.upper().startswith("DEVELOP"):
```

Comment:
I would put this in the and above instead of a new line.

I.e., `if gem5_version and not gem5_version.upper().startswith("DEVELOP") and not self._...`

Reply:
In the interests of getting this in, I'll let this slide. Even though it's inelegant, it's still understandable.